project/src/components/layout/layout.tsx

```diff
@@ -22,7 +24,7 @@ function Layout(): JSX.Element {
 
   return (
     <div className={clsx('page',
-      {'page--gray page--main': location.pathname === AppRoute.Root},
+      {'page--gray page--main': location.pathname === AppRoute.Root || matchCityRoute !== null},
```

Comment:
`matchCityRoute ` и так вернут null

Reply:
```suggestion
      {'page--gray page--main': location.pathname === AppRoute.Root || matchCityRoute},
```
вот так исправила. мне нужны эти классы для root и страниц городов

Reply:
так вполне рабочий вариант)